pyproject.toml

```
@@ -119,6 +119,22 @@ testpaths = "tests piptools"
 filterwarnings = ["always"]
 markers = ["network: mark tests that require internet access"]
 
+[tool.ruff]
+line-length = 100
+
+[tool.ruff.lint]
+extend-select = [
+  "PT001",
+  "PT006",
+  "PT007",
+  "PT023",
```

Just enable them all.
```suggestion
  "PT",
```